waspc/data/Generator/templates/sdk/wasp/server/dbClient.ts

```diff
@@ -1,10 +1,17 @@
 {{={= =}=}}
 {=# areThereAnyEntitiesDefined =}
-import Prisma from '@prisma/client'
-
-function createDbClient(): Prisma.PrismaClient {
-  return new Prisma.PrismaClient()
+{=# prismaSetupFn.isDefined =}
+{=& prismaSetupFn.importStatement =}
+function createDbClient() {
+  return {= prismaSetupFn.importIdentifier =}()
+}
+{=/ prismaSetupFn.isDefined =}
+{=^ prismaSetupFn.isDefined =}
+import { PrismaClient as InternalPrismaClient } from '@prisma/client'
+function createDbClient() {
```

Comment:
Another empty line mising here. No big deal, but it's easy to keep tidy so why not.